### DISCUSSION TITLE
Bug in the Store function

### DISCUSSION THREAD

**Discussion by Aizahkmayo** [Cyfrin/foundry-full-course-cu#1915] [Q&A] [answered]
<img width="881" alt="Screenshot of store function" src="https://github.com/Cyfrin/foundry-full-course-cu/assets/160310354/63477d7b-927a-4820-a253-4772ef98151a">
How can i solve this issue ?

**Comment by EngrPips** [comment, marked as answer]
Hello @Aizahkmayo, change your code to the code below, and it should work.

```js
 favouritenumber = _favouritenumber;
```

**Comment by Aizahkmayo** [comment]
Thanku soo much...

**Reply by EngrPips** [reply]
You are welcome. Friend. Keep crushing it.

**Comment by Aizahkmayo** [comment]
Thanks, I will.
